mypy/plugin.py

```diff
@@ -692,9 +692,32 @@ def get_class_decorator_hook(self, fullname: str
 
         The plugin can modify a TypeInfo _in place_ (for example add some generated
         methods to the symbol table). This hook is called after the class body was
-        semantically analyzed.
+        semantically analyzed, but *there may still be placeholders*.
```

Comment:
Maybe briefly mention that placeholders are caused by forward refs (plugin devs read this file as docs).